### PR TITLE
remote: ssh: fix error message

### LINKED ISSUE #1996: ssh: fix exception reporting during upload
See https://github.com/iterative/dvc/blob/0.40.1/dvc/remote/ssh/__init__.py#L237

should be:
```
                        "failed to upload '{}' to '{}'".format(from_info, to_info)
```

Discord https://discordapp.com/channels/485586884165107732/485596304961962003/577533105611735051

### PR DESCRIPTION
**Pull request by kurianbenoy**
* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [x] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

Fixes #1996

### CODE REVIEW

**Comment by efiop** [comment]
Hi @kurianbenoy ! Looks like your code doesn't pass format checks. :slightly_smiling_face:  You probably forgot to install git hook as per https://dvc.org/doc/user-guide/contributing . Please run these commands to format and check that everything is fine:
```
black dvc/
black tests/
flake8 dvc/
flake8 tests/
```

**Comment by kurianbenoy** [comment]
Can I rectify this error by having proper code with no flake 8 erros. I guess I am having some issues with pre-commit hook

**Comment by kurianbenoy** [comment]
@efiop review when you are free.

File: dvc/remote/ssh/__init__.py [hunk 1]
```diff
@@ -234,13 +234,11 @@ def upload(self, from_infos, to_infos, names=None, no_progress_bar=False):
...
-                        "failed to upload '{host}/{path}' to '{dest}'".format(
-                            host=from_info.host,
-                            path=from_info.path,
-                            dest=to_info.path,
+                        "failed to upload '{}' to '{}'".format(
+                            from_info, to_info
                         )
                     )
-                    pass
+                pass
```
Comment (efiop) [inline dvc/remote/ssh/__init__.py]:
```suggestion
```

Comment (efiop) [reply dvc/remote/ssh/__init__.py]: Let's also remove this one while we are at it :)

**Pull request merged**